djpsa/sync/management/commands/base_callbacks_deleted.py

```diff
@@ -0,0 +1,15 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from djpsa.api.exceptions import APIClientError
+
+
+class BaseDeleteCommand(BaseCommand):
+    help = 'Ensure callbacks are deleted on ConnectWise.'
```

Comment:
ConnectWise shouldn't be mentioned, here and 2 other places.

Reply:
oof